microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonMethodExecutor.java

```diff
@@ -99,9 +124,9 @@ private void jUnitTestNameRule(TestMethodExecutor testMethodExecutor) {
                         Field nameField = TestName.class.getDeclaredField("name");
                         nameField.setAccessible(true);
                         nameField.set(testName,
-                                Description.createTestDescription(
-                                        instance.getClass(),
-                                        testMethodExecutor.getMethod().getName()).getMethodName());
+                                      Description.createTestDescription(
```

Comment:
you should not reformat existing code as part of other PRs - there is no change here, so it makes it harder to review